evhtp.c

```diff
@@ -2527,7 +2547,12 @@ htp__connection_eventcb_(struct bufferevent * bev, short events, void * arg)
              * but libevent will disable the read side of the bufferevent
              * anyway, so we must re-enable it.
              */
-            bufferevent_enable(bev, EV_READ);
+            log_debug("errno EAGAIN");
```

Comment:
So there is a patch request in for libevent to make this not happen. For some reason libevent tries to accept() in a loop, then stalls the socket at the second attempt.